cmd/soroban-rpc/internal/methods/get_events.go

```diff
@@ -510,6 +512,7 @@ func (h eventsRPCHandler) getEvents(ctx context.Context, request GetEventsReques
 	return GetEventsResponse{
 		LatestLedger: ledgerRange.LastLedger.Sequence,
```

Comment:
will the latest ledger here be in sync with the latest ledger available in `h.dbReader.GetEvents()`? it seems that there is a race condition where a ledger is ingested after `h.ledgerReader.GetLedgerRange(ctx)` but before `h.ledgerReader.GetLedgerRange(ctx)`

Reply:
We talked about this offline in detail but TLDR here is that we set the (startLedger, endLedger) first and pass it to DB so there is no race condition